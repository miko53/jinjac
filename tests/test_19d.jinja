{% if not_exist %}
{% if a_ident %}
  --> shoud not display '{{ a_ident }}'
{% else %}
  --> shoud not display '{{ a_ident }}' too
{% endif %}
{% else %}
{% if a_ident %}
  --> shoud display '{{ a_ident }}' here
{% else %}
  --> shoud not display '{{ a_ident }}' in this case
{% endif %}
{% endif %}
TOTO
