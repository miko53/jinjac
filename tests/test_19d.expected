

  --> shoud display 'TheValeur' here


TOTO
